OpenOversight/app/templates/cop_face.html

```diff
@@ -42,104 +43,98 @@ <h2>
           </div>
           <div class="col-sm-3 text-center">
             <div class="docs-data">
-              {% if department %}
-                <form action="{{ url_for('main.label_data', image_id=image.id, department_id=department.id) }}"
-                      method="post">
-                </form>
-              {% else %}
-                <form action="{{ url_for('main.label_data', image_id=image.id) }}"
-                      method="post">
-                </form>
-              {% endif %}
-              {{ form.hidden_tag() }}
-              <div class="input-group input-group-sm">
-                <input type="text"
-                       class="hidden form-control"
-                       name="dataX"
-                       id="dataX"
-                       placeholder="x">
-              </div>
-              {% for error in form.dataX.errors %}
-                <p>
-                  <span style="color: red;">[{{ error }}]</span>
-                </p>
-              {% endfor %}
-              <div class="input-group input-group-sm">
-                <input type="text"
-                       class="hidden form-control"
-                       name="dataY"
-                       id="dataY"
-                       placeholder="y">
-              </div>
-              <div class="input-group input-group-sm"></div>
-              {% for error in form.dataY.errors %}
-                <p>
-                  <span style="color: red;">[{{ error }}]</span>
-                </p>
-              {% endfor %}
-              <div class="input-group input-group-sm">
-                <input type="text"
-                       class="hidden form-control"
-                       name="dataWidth"
-                       id="dataWidth"
-                       placeholder="width">
-              </div>
-              {% for error in form.dataWidth.errors %}
-                <p>
-                  <span style="color: red;">[{{ error }}]</span>
-                </p>
-              {% endfor %}
-              <div class="input-group input-group-sm">
+              <form action="{{ url_for('main.label_data', image_id=image.id, department_id=department.id if department else 0) }}"
+                    method="post">
```

Comment:
This should address the error, @sea-kelp: https://github.com/lucyparsons/OpenOversight/pull/968#discussion_r1301347096

Reply:
<img width="872" alt="Screenshot 2023-08-23 at 3 16 03 PM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/67a25f43-88f4-4b08-ab97-8f6ac918a63d">

<img width="659" alt="Screenshot 2023-08-23 at 3 17 37 PM" src="https://github.com/lucyparsons/OpenOversight/assets/5885605/cf15278d-17b5-4052-92ad-7c4c7163997f">